packages/perseus/src/widgets/radio/widget.jsx

```diff
@@ -40,27 +45,41 @@ type UserInput = {|
 type Rubric = PerseusRadioWidgetOptions;
 type Props = WidgetProps<RenderProps, Rubric>;
 
+type DefaultProps = {|
+    choices: Props["choices"],
+    multipleSelect: Props["multipleSelect"],
+    countChoices: Props["countChoices"],
+    deselectEnabled: Props["deselectEnabled"],
+    linterContext: Props["linterContext"],
+|};
+
 class Radio extends React.Component<Props> {
     focusFunction: FocusFunction;
 
-    static defaultProps: $FlowFixMe = {
-        choices: [{}],
-        displayCount: null,
+    static defaultProps: DefaultProps = {
+        choices: [],
         multipleSelect: false,
         countChoices: false,
         deselectEnabled: false,
         linterContext: linterContextDefault,
     };
 
-    _renderRenderer: (string) => React.Node = (content) => {
+    _renderRenderer: (content?: string) => React.Node = (
+        content?: string = "",
+    ) => {
         content = content || "";
```

Comment:
This seems redundant.
```suggestion
    _renderRenderer: (content?: string) => React.Node = (
        content?: string = "",
    ) => {
```

Reply:
Will update this. It was removed in Kevin's example, but it looks like I missed it.

So `content = content || ""` means the same thing as passing in `content?: string = ""`?